app/pages/project/home/home-workflow-button.jsx

```diff
@@ -77,4 +98,16 @@ ProjectHomeWorkflowButton.propTypes = {
     id: PropTypes.string
   }).isRequired,
   workflowAssignment: PropTypes.bool
-};
\ No newline at end of file
+};
+
+const mapStateToProps = state => ({});
+
+const mapDispatchToProps = dispatch => ({
+  actions: {
+    classifier: bindActionCreators(classifierActions, dispatch)
+  }
+});
+
+export default connect(mapStateToProps, mapDispatchToProps)(ProjectHomeWorkflowButton);
+export { ProjectHomeWorkflowButton };
```

Comment:
What is the purpose of having two exports here? (`export default` on line 111 and `export` on line 112)

Reply:
The unwrapped component is exported for tests.